main.nf

```diff
@@ -1,102 +1,163 @@
 #!/usr/bin/env nextflow
 
-/// To use DSL-2 will need to include this
-nextflow.enable.dsl=2
 
 // =================================================================
 // main.nf is the pipeline script for a nextflow pipeline
-// Should contain the following sections:
-	// Process definitions
-    // Channel definitions
-    // Workflow structure
-	// Workflow summary logs 
-
-// Examples are included for each section. Remove them and replace
-// with project-specific code. For more information see:
-// https://www.nextflow.io/docs/latest/index.html.
-//
 // ===================================================================
 
-// Import processes or subworkflows to be run in the workflow
-// Each of these is a separate .nf script saved in modules/ directory
-// See https://training.nextflow.io/basic_training/modules/#importing-modules 
-include { processOne } from './modules/process1'
-include { processTwo } from './modules/process2' 
 
-// Print a header for your pipeline 
+// Import subworkflows to be run in the workflow
+include { checkInputs                                } from './modules/checkCohort'
+include { mutect2                                    } from './modules/mutect2'
+include { GatherVcfs                            } from './modules/GatherVcfs'
+include { MergeMutectStats                           } from './modules/MergeMutectStats'
+include { LearnReadOrientationModel                  } from './modules/LearnReadOrientationModel'
+include { GetPileupSummaries_T; GetPileupSummaries_N } from './modules/GetPileupSummaries'
+include{  CalculateContamination                     } from './modules/CalculateContamination'
+include { FilterMutectCalls                          } from './modules/FilterMutectCalls'
+include { getFilteredVariants                        } from './modules/getFilteredVariants'
+//include { annotateWithSnpEff                        } from './modules/annotateWithSnpEff'
```

Comment:
Is this in use or still to be added? When running the workflow, I got the following message: 
```
WARN: There's no process matching config selector: annotateWithSnpEff 
```

Reply:
I am removing snpEff related process for now. There are some additional requirements with regards to - downloading the snpEff database etc (by the user). Need clarity about whats the best approach. I will finalsie this in next update